brave/browser-laptop
# Refactor switchControl with Aphrodite

Closes #11065
Closes #7321

- Remove switchControls.less 🎉
- Replace `className` with `customWrapper`
- Add color properties to `theme.js`
- Update data-test-id

Regression Test:
- Make sure https://github.com/brave/browser-laptop/issues/11937#issue-273570256 does not regress

Test Plan:
1. Open http://example.com
2. Click the shield icon
3. Make sure the switches work

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
